addons/hr_work_entry_contract/static/src/js/work_entries_controller_mixin_owl.js

```diff
@@ -0,0 +1,46 @@
+/** @odoo-module **/
```

Comment:
This one would be `view/hooks.js` or `views/work_entry_hook.js` if you wanna be more specific.